### PR TITLE
Add unique constraint for owner_id and owner_type on Subscriptions

### PR DESCRIPTION
**Pull request by anarosas**

### CODE REVIEW

File: db/main/migrate/20170203124000_add_constraint_for_owner_id_to_subscriptions.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddConstraintForOwnerIdToSubscriptions < ActiveRecord::Migration
+  def up
+    execute "ALTER TABLE subscriptions ADD CONSTRAINT subscriptions_owner_id UNIQUE (owner_id, owner_type);"
```
Comment (renee-travisci) [inline db/main/migrate/20170203124000_add_constraint_for_owner_id_to_subscriptions.rb]: The constraint should be unique owner_id and owner_typer where status == 'subscribed' to match the rails constraint.

**Comment by anarosas** [comment]
@renee-travisci I made the changes to add the WHERE clause, could you give it a look please? Also could you please check if the `structure` file looks correct? thank you!

**Comment by anarosas** [comment]
We should probably fix this before adding this constraint: https://github.com/travis-pro/team-teal/issues/1575

**Comment by anarosas** [comment]
@renee-travisci yes, I'm looking into the existing doubled subscriptions

**Pull request merged**